lang/csharp/src/apache/codegen/AvroGen.cs

```diff
@@ -142,7 +141,8 @@ static void Usage()
                 AppDomain.CurrentDomain.FriendlyName);
             return;
         }
-        static int GenProtocol(string infile, string outdir,
```

Comment:
Is it technically possible this method to be used without going thru `Main()` in 1.11.0 ?